### PR TITLE
Fixes #642 Hero Banner

### LINKED ISSUE #642: Make a Hero Banner
**What would you like to be added**: [A Hero Banner](https://www.tributemedia.com/blog/what-is-a-hero-banner)

**Why would you like this to be added**: Based on our [discussions](https://github.com/orgs/Seneca-CDOT/teams/telescope-frontend/discussions/1) about the design of the front-end, we have decided to include a Hero banner for Telescope. as it is a great way to quickly give the user an idea what Telescope is about and what they can do.

The image we want is this: 

image

And this is how the banner should look like:
image

### PR DESCRIPTION
**Pull request by cindyledev**
## Issue This PR Addresses
Fixes #642 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Just throwing this out there because CSS makes my knees weak and I'm half tempted to photoshop the entire banner into this. I have failed you @agarcia-caicedo :(

Problems I've encountered:
- can't get gradient to work on top of the image
- button has a box around it
- can't move button to the bottom of the image
- the last e letter of Telescope doesn't sit on top of the sun
- lots of internal crying

Possible solutions/next steps:
- image may need to be responsive to accommodate different screen sizes (large laptop, small laptop, tablet, phone). Like photoshop 4 different image sizes for each screen?
- text and button will need to be responsive as well
- button should have scroll effect to scroll down to the blog portion, button currently alerts "Help"
- Does React have a Bootstrap thing so I don't have to manually CSS everything?

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by agarcia-caicedo** [comment]
Instead of making the image a background image, you should probably set the background to a dark colour and add opacity to the image

**Comment by raygervais** [comment]
Hey @cindyledev, I'd be happy to help you address some of the problems you listed above in either a follow-up PR, or as part of this one (up to you and @humphd to decide) if you're interested in getting a second pair of eyes on it.

**Comment by cindyledev** [comment]
@agarcia-caicedo you're some kind of CSS wizard, I'm making progress for the banner. I had my logic all backwards, your solution worked. Thanks a lot :)

**Comment by cindyledev** [comment]
Updated and rebased this PR, it's ready for review now

File: src/frontend/src/components/Banner/Banner.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.bannerImg {
+  background-image: url(../../images/hero-banner.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: scroll;
+  background-size: cover;
+  height: 100vh;
+  opacity: 0.7;
```
Comment (agarcia-caicedo) [inline src/frontend/src/components/Banner/Banner.css]: Slight nit, could you change the opacity anywhere between 0.3-0.5? The image is a little too bright

**Comment by cindyledev** [comment]
I've made the opacity changes

**Pull request merged**